pkg/apigateway/main.go

```diff
@@ -0,0 +1,191 @@
+// Copyright 2018 The OpenPitrix Authors. All rights reserved.
```

Comment:
I think mv pkg/apigateway/main.go to pkg/services/notification/api_gateway.go will be better.
Add apiGateway.Serve() in Serve() of pkg/services/notification/server.go.